python/deptry/dependency_getter/pep_621.py

```diff
@@ -15,6 +15,7 @@
 
 @dataclass
 class PEP621DependencyGetter(DependencyGetter):
+    pep621_dev_dependency_groups: tuple[str, ...] = ()
     """
     Class to extract dependencies from a pyproject.toml file in which dependencies are specified according to PEP 621. For example:
```

Comment:
The note at the end of the docstring below this, I believe, is invalidated by this change.

Reply:
Well spotted, thanks!